### PR TITLE
Fixes for trackable

### PR DESCRIPTION
**Pull request by maier49**
**Type:** bug and feature

**Description:** 

Leverages the existing `fetchAroundUpdates` behavior in the observable mixin for updating tracked items. Adds checks for items that have been pushed into or out of the queried range of items, but were not directly updated.

**Related Issue:** #28 

Please review this checklist before submitting your PR:
- [x] There is a related issue
- [x] All contributors have signed a CLA
- [ ] All code matches the [style guide](https://github.com/dojo/meta/blob/master/STYLE.md)
- [x] The code passes the CI tests
- [x] Unit or Functional tests are included in the PR
- [x] The PR increases or maintains the overall unit test coverage percentage
- [x] The code is ready to be merged

### CODE REVIEW

**Comment by kitsonk** [comment]
I would like to see the code coverage for this when CI completes before I fully review.

**Comment by maier49** [comment]
@kitsonk CI finished. Had to push up a dummy commit as the DNS issues last Friday seem to have broken that build.

File: src/store/mixins/createOrderedOperationMixin.ts [hunk 1]
```diff
@@ -64,6 +64,10 @@ function queueStoreOperation(operation: Function, returnsPromise?: boolean) {
 					});
 				});
 			});
+			(<any> promise).executeNext = function() {
```
Comment (agubler) [inline src/store/mixins/createOrderedOperationMixin.ts]: curious about this?

Comment (maier49) [reply src/store/mixins/createOrderedOperationMixin.ts]: That was the start of a bad idea that I was toying with that should not have been committed! I removed it.

Comment (agubler) [reply src/store/mixins/createOrderedOperationMixin.ts]: Cool!

**Review by kitsonk — APPROVED** [review]
👍 🎃

**Pull request merged**